### PR TITLE
refactor(devtools-view): Remove dependency on and usages of legacy FluentUI package

### PR DESCRIPTION
**Pull request by Josmithr**
Also makes some some misc. styling updates to better match new library's intended styling mechanisms.

Fixes a number of overflow issues, adding scrollbars where needed.

image

image

image

image

### CODE REVIEW

File: packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx [hunk 1]
```diff
@@ -197,12 +216,20 @@ interface ListLengthSelectionProps {
 	onChangeSelection(newLimit: number): void;
 }
 
+const useListLengthSelectionStyles = makeStyles({
+	root: {
+		// Inject 20px gap between each item
+		// ...shorthands.gap("20px"),
```
Comment (Josmithr) [inline packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx:222]: TODO: check this.

File: packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx [hunk 2]
```diff
@@ -254,7 +281,7 @@ interface FilteredTelemetryViewProps {
 	 */
 	setIndex: React.Dispatch<React.SetStateAction<number | undefined>>;
 	/**
```
Comment (Josmithr) [inline packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx:283]:
```suggestion

	/**
```

File: packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx [hunk 3]
```diff
@@ -508,21 +535,43 @@ function FilteredTelemetryView(props: FilteredTelemetryViewProps): React.ReactEl
...
+			pane1Style={{ overflowY: "auto" }}
+			pane2Style={{ margin: "10px" }}
+			resizerStyle={{
+				borderRight: `2px solid ${tokens.colorNeutralForeground2}`,
+				borderLeft: `2px solid ${tokens.colorNeutralForeground2}`,
+				zIndex: 1,
+				cursor: "col-resize",
+			}}
+		>
+			<DataGrid
```
Comment (Josmithr) [inline packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx:554]: Where did this come from? Remove.

File: packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx [hunk 4]
```diff
@@ -42,14 +43,34 @@ import { Waiting } from "./Waiting";
  */
 const DEFAULT_PAGE_SIZE = 100;
 
+const useTelemetryViewStyles = makeStyles({
+	root: {
+		alignItems: "start",
+		display: "flex",
+		flexDirection: "column",
+		// justifyItems: "start",
```
Comment (Josmithr) [inline packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx:51]: TODO: clean up

File: packages/tools/devtools/devtools-view/src/components/Menu.tsx [hunk 5]
```diff
@@ -47,43 +62,48 @@ export interface MenuItemProps {
 	onClick: (event: unknown) => void;
 	text: string;
 	isActive: boolean;
+
+	/**
+	 * The icon to display in the header of the menu section.
+	 */
+	icon?: React.ReactElement;
```
Comment (Josmithr) [inline packages/tools/devtools/devtools-view/src/components/Menu.tsx:69]: Remove

File: packages/tools/devtools/devtools-view/src/components/ContainerSummaryView.tsx [hunk 6]
```diff
@@ -353,6 +365,14 @@ export interface IContainerActions {
 	closeContainer?: () => void;
 }
 
+const useActionBarStyles = makeStyles({
+	root: {
+		...shorthands.margin("5px"), // TODO: verify this
```
Comment (Josmithr) [inline packages/tools/devtools/devtools-view/src/components/ContainerSummaryView.tsx:370]: TODO

**Review by alexvy86 — APPROVED** [review]

**Review by alexvy86 — APPROVED** [review]

**Pull request merged**